packages/app/src/search/SearchTab.module.css

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.inputRow {
+  display: flex;
+}
+
+.inputRow input {
+  border-radius: 0.5rem;
+  margin: 0.25rem;
+  box-shadow: none;
+  height: 1.875rem;
+}
```

Comment:
Current aspect:

![image](https://user-images.githubusercontent.com/2936402/226878573-845f0f50-396b-4e51-9007-92a50710457b.png)

Nitpicking here, but it'd be good, I think, if the search box:
- had a bit more space all around it
- had a bit more padding inside it
- filled the sidebar in width

... and if the total height of `.inputRow` matched the height of the explorer's root group, so that the first search result and first child entity line up perfectly.

Reply:
The results could also use a bit of horizontal padding -- it should probably match the horizontal padding of the explorer's root group.

Reply:
I updated the aspect according to your comments

Reply:
Looks great! :star_struck: